core/client-internal/src/main/java/alluxio/client/UfsUtils.java

```diff
@@ -42,177 +40,70 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
   /**
-   * Builds a new path relative to a given root by retrieving the given path relative to the
-   * ufsRootPath.
+   * Mounts the given folder in the under storage system to an Alluxio path which does not exist.
+   * Then loads metadata for each file in the under file system unless excluded by the exclusions
+   * list. If the folder has already been mounted, this method assumes the mount is correct and
+   * will continue to load metadata for each file. This method can be called repeatedly to sync
```

Comment:
I think "sync" is not well defined here and might be misleading. For instance, if something is deleted in UFS, it will not be removed from Alluxio. My understanding is that calls to loadUfs will load all files and folders that do not already exist in Alluxio, is that correct?